### PR TITLE
Add Pimcore and MacPass

### PR DESCRIPTION
**Pull request by dpfaffenbauer**

### CODE REVIEW

**Comment by Kickball** [comment]
Would you mind putting them in alphabetical order please?

File: README.md [hunk 1]
```diff
@@ -397,6 +399,7 @@ CMS are a practical way to setup a website with many features. CMS often come wi
  * [Keepass](http://keepass.info) - Free, open source, light-weight and easy-to-use password manager. `GPLv2` `C#`
  * [Rattic](http://rattic.org/) - Password Management for Humans ([Source code](https://github.com/tildaslash/RatticWeb))
  * [Teampass](http://www.teampass.net/) - Password manager dedicated for managing passwords in a collaborative way on any server Apache, MySQL and PHP. ([Source code](https://github.com/nilsteampassnet/TeamPass/)) `GPLv3` `PHP`
+ * [MacPass](https://github.com/mstarke/MacPass) - OSX Port of KeePass  ([Source code](https://github.com/mstarke/MacPass)) `GPLv3` `ObjectiveC`
```
Comment (nodiscc) [inline README.md]: This is not server-side/not a network service, see https://github.com/Kickball/awesome-selfhosted/pull/43. @dpfaffenbauer can you remove it?

Comment (dpfaffenbauer) [inline README.md]: nor is keepass.info ?

**Comment by dpfaffenbauer** [comment]
Order has been changed

File: README.md (hunk 1 above)
Comment (nodiscc) [inline README.md]: You are right. Keepass should be removed.

**Comment by dpfaffenbauer** [comment]
Removed MacPass and Keepass (Keepass is also not Hosted)

**Comment by nodiscc** [comment]
Thanks!

**Comment by Kickball** [comment]
If the files changed page is displaying properly then you have moved Pico's License to the end of Pimcore's line.

Could you fix that before we merge please.

**Comment by dpfaffenbauer** [comment]
oh yeah sure, done.

**Pull request merged**